srcpkgs/akonadi5/template

```diff
@@ -16,7 +16,7 @@ maintainer="Helmut Pozimski <helmut@pozimski.eu>"
 license="LGPL-2.1-or-later"
 homepage="https://community.kde.org/KDE_PIM/Akonadi"
 distfiles="${KDE_SITE}/applications/${version}/src/akonadi-${version}.tar.xz"
-checksum=f930deaade1cac1488b8af05cc28f4a78a441c34dbe875b673af3423f8a14658
+checksum=c0a2c4259b51293d21487cfe2295df6e8dcfb7ab50eb1a698f531dc7d3253e9a
 conflicts="akonadi"
```

Comment:
I think this can be dropped, no ?

Reply:
If there are no Void installations around somewhere that still use KDE 4, it can be.